tests/test_meteodatalab/test_intpl_k2any.py

```diff
@@ -4,12 +4,25 @@
 # First-party
 from meteodatalab.grib_decoder import GribReader
 from meteodatalab.operators.destagger import destagger
-from meteodatalab.operators.vertical_interpolation import interpolate_k2any
+from meteodatalab.operators.vertical_interpolation import (
+    TargetCoordinates,
+    TargetCoordinatesAttrs,
+    interpolate_k2any,
+)
 
 
 def test_intpl_k2theta(data_dir, fieldextra):
     # define target coordinates
-    tc_values = [15.0]
+    tc = TargetCoordinates(
```

Comment:
two aside notes: I think the name of the function is wrong here ? 
And wouldnt be better if test intpl_k2any with more than 1 target values ? The positive="up" or "down" I guess it has no effect... You could create an issue for that if you agree?

Reply:
Yes it is wrong and I'm not sure what the intent of the positive up or down was, perhaps it is supposed to end up in the GRIB metadata somehow. I can try to find another test case that uses interpolate k2any in fieldextra.

Reply:
unless you are doing this in this PR, can you create a new jira task so that we do not forget this ?

Reply:
I searched but I can't find an example where intpl_k2any is called with multiple values at once. The operator is called through the `find_value` vertical operator and it seems that only one value can be supplied at a time. However, the are some examples where the operator is called multiple times. Should I use that?